app/jobs/rest_of_world_zones/rest_of_world_zone_update_job.rb

```diff
@@ -0,0 +1,55 @@
+module RestOfWorldZones
+  class RestOfWorldZoneUpdateJob < ApplicationJob
```

Comment:
please, try to combine `rest_of_world_zone_create_job.rb ` and `rest_of_world_zone_update_job.rb` into one job, I checked it quickly and I see that they're very similar.
If it's not possible to use just one job instead of 2, please extract the same code in separate service object.

Reply:
I thought about it, but for me, there are too many differences.